tests/test_metrics.py

```diff
@@ -115,13 +115,13 @@ def check_metrics(self, Y, Y_pred, expected, span_type=None):
         macro_f1 = sequence_f1(Y, Y_pred, span_type=span_type, average="macro")
         for cls_ in counts:
             for metric in counts[cls_]:
-                self.assertEqual(len(counts[cls_][metric]), expected[cls_][metric])
-            self.assertAlmostEqual(recalls[cls_], expected[cls_]["recall"], places=3)
+                self.assertEqual(len(counts[cls_][metric]), expected[cls_][metric], msg=f"class={cls_}, metric={metric}")
+            self.assertAlmostEqual(recalls[cls_], expected[cls_]["recall"], places=3, msg=f"class={cls_}")
```

Comment:
Why can't this be equal?

Reply:
I think it's probably good practice to use almostEqual in place of Equal when doing floating point equality checks.